### DISCUSSION TITLE
Hi, my account has been flagged.

### DISCUSSION THREAD

**Discussion by pgupta500** [community/community#101744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey! As the title suggests, i think that my account has been flagged. Can you please unflag it and let me know what can be done by me in order to prevent it in the future?

**Comment by samus-aran** [comment]
Hi @Coder321P ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
